Makefile

```diff
@@ -132,8 +132,6 @@ endif
 	cd upstream-tools && yarn install --frozen-lockfile
 	# Apply all automated changes
 	cd upstream-tools && yarn --silent run apply
-	# Check for any pending replacements
```

Comment:
If not enabled the checks would be failing right? Skipping this doesn't forgo any validation we had before? Just double-checking.

Reply:
We do not forgo any "validation" that we used to have. `yarn run check` looked for instances of `[tT]erraform` still remaining in the upstream docs, and added them as TODOs to `replacements.json`. This is now done in `PostTfgenHook`.

This step never failed, and still doesn't, so I wouldn't really call it validation.

Reply:
Sounds good. Just making sure since I didn't stop to read what it did but you probably read it. Thank you! All good then.